jina/executors/__init__.py

```diff
@@ -368,7 +368,7 @@ def get_shard_workspace(
 
         :return: returns the workspace of the shard of this Executor.
         """
-        if replica_id == -1 or replica_id is None:
+        if replica_id == -1 or replica_id is None or replica_id == 'None':
```

Comment:
why is it even considered? it should never be a string right?

Reply:
yes, I'm still investigating why there is a 'None'- string in some cases. It should not be required.

Reply:
okay done